.prettierignore

```diff
@@ -1,2 +0,0 @@
-build
-dist
```

Comment:
This was removed by the migrator provided by Capacitor and wasn't running into issues even after being removed and building so I left it out.